spec/lib/miq_automation_engine/miq_ae_service_spec.rb

```diff
@@ -102,6 +102,14 @@ module MiqAeServiceSpec
       it "expose all expected active_record models as service_models" do
         excluded_model_names = %w(
           AuthToken
+          AuthenticationAllowAll
+          AuthenticationGithub
+          AuthenticationGoogle
+          AuthenticationHtpasswd
+          AuthenticationLdap
+          AuthenticationOpenId
+          AuthenticationRequestHeader
+          AuthenticationRhsm
```

Comment:
I'm not sure why Rhsm was added to the list, but there was a model built for it anyway.